catalyst_voices/lib/widgets/buttons/voices_text_button.dart

```diff
@@ -2,7 +2,7 @@ import 'package:catalyst_voices/widgets/buttons/voices_button_affix_decoration.d
 import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
 import 'package:flutter/material.dart';
 
-enum _Variant { primary, neutral, secondary }
+enum _Variant { primary, neutral, secondary, customColor }
```

Comment:
Just `custom`? we have `primary`, not `primaryColor`

Reply:
Here, we are using `iconsError` Color. It's good to have the possibility to use custom color for the button in case like that, isn't it?

Reply:
Doesn't matter which color is used, `customColor` name seems not aligned with other variant names.

When these variants were created somebody choose to call them in a certain way, lets stick to it unless there is some argument to not follow it.

Reply:
thx for the clarification